repository_service_tuf_worker/services/keyvault/local.py

```diff
@@ -17,6 +20,13 @@
 from repository_service_tuf_worker.interfaces import IKeyVault, ServiceSettings
 
 
+@dataclass
+class LocalKey:
+    filename: str
+    password: str
+    type: Optional[str] = "ed25519"
```

Comment:
Can you also add an optional key name as this was already added in the CLI and the API?